tests/test_port_an.py

```diff
@@ -6,6 +6,29 @@
 
 
 class TestPortAutoNeg(object):
+    def test_PortAutoNegForce(self, dvs, testlog):
+
+        db = swsscommon.DBConnector(0, dvs.redis_sock, 0)
+
```

Comment:
should you prepare the adb now as you did on the prev test for fec? any reason here or there it is not the same flow?

Reply:
It is initialized in the flow lines below.

Reply:
I know
I was asking if you should have the 2 DBs unit just after each other like you did in the prev function just to have the same flow

Reply:
Got you. Will move the initialization to one place.